Organization (#115)
bump-sh/docs: src/_help/organizations/create-and-manage-organizations.md

---
title: Create and manage organizations
---

- TOC
{:toc}

## How to create an organization
To create an organization, you should start by accessing the settings of your personal account.

> The Organizations section allows you to view the organizations you are currently associated with and create a new one.

Author: 👍

![](/images/help/create-org.png)

You will then be prompted to specify the organization's name (visible to all its members and guests) and its slug, which determines its URL (in the format bump.sh/your-organization-slug/).

![](/images/help/org-creation.png)

> At this stage, you can also choose to add documentation and/or hubs to your new organization. These will retain their access settings as they had before the migration.

Reviewer: ```suggestion
At this stage, you can also choose to add existing documentation and/or hubs to your new organization. These will retain the access settings they had before the migration.
```

> At this stage, you can also choose to add existing documentation and/or hubs to your new organization. These will retain the access settings they had before the migration.

Author: 👍

## Add teammates to an organization
From the Members section in the settings of an organization, you can add new members, view the list of existing members, modify their roles, and check pending invitations.

To invite a new member, simply provide their email address and specify the role you wish to assign to them.

![](/images/help/org-add-member.png)

[Roles](/help/organizations/organization-access-management/#roles) allow you to precisely adjust what a member of your organization can or cannot do. You can change a member's role at any time.

> Please note that only the owner of the organization has access to billing and its options.
{: .info}

### Add teammates with SSO

> The SSO support allows you to add and manage members of your organization directly through your existing solution. We support most of the available identity providers (list them), through WorkOS.

Reviewer: I think you forgot to list the identity providers :P

Author: 👍

> Using SSO does not prevent you from inviting users via manual invitations.
{: .info}

> SSO is a feature available from our Business plan.
{: .info}

## Delete teammates in an organization
Still from the Members section, you can easily remove a member from your organization by clicking on their role to the right of their name and email. The option to delete this member is available at the very end of the list that appears.

## Change URL/slug
From the settings of your organization, you can quickly change the slug via the dedicated field.

> Bump.sh does not automatically redirect to the new URL after changing the slug, so be sure to notify your users of this change.

Reviewer 2: I believe we should give a bit more context around the slug of an organisation. Which can also be thought of a “namespace”.

We should warn the users that this slug is used for resources created within their organisation. I.e. docs and hubs created under an organisation will be available on a URL that is “below” their organisation slug (Apart from resources with a custom domain).

> The Custom Domain option allows you to keep your users within your brand experience by fully customizing the URL of your organization.

Reviewer: What do you think of adding a link to the Custom Domain page?

Reviewer 2: We don't give the possibility to use a custom domain on an organisation unfortunately. Custom domains can only be set on a Hub, or on a Documentation.

Author: My bad, thanks for clarifying this 🙇

> Modifying the slug also changes all the public URLs for accessing your organization, hubs, and documentation!
{: .warning}

> ## Transfer Ownership

Reviewer 3: Can we please be consistent with carriage returns? Always addding one before and after titles seems to be a good rule.

Transferring ownership is mandatory if you want to leave an organization. You can only transfer ownership to a member with admin role.

> You can transfer the ownership of your organization by contacting us directly.

Reviewer 3: I think te sentence could be a bit easier to understand by starting by the goal. Something like "As the owner of an organization, before leaving this one you need to transfer your ownership to another member. This can be done by contacting us."

Author: Is leaving the organization the only reason you want to transfer the ownership?

> If you choose to close your Bump account but are still the owner of an organization, a warning message will be displayed.

Reviewer 3: ```suggestion
If you choose to close your Bump.sh account but are still the owner of an organization, a warning message will be displayed.
```

Please consider transferring ownership if necessary.

## Delete an organization (and consequences on existing docs)

> This action can be initiated from the organization's settings, in the danger zone.

Reviewer 3: Could we add more warning about the fact that this should be done VERY carrefully, in very specific situations?

Author: Not 100% sure of that part to be honest but I've tried something in the next commit.

![](/images/help/org-danger-zone.png)

> > Deleting an organization results in its permanent and irreversible erasure: the documentation and hubs it contains will also be destroyed.

Reviewer: It feels a bit weird, what do you think of a simpler version?
```suggestion
> Deleting an organization is permanent and irreversible; the documentation and hubs it contains will also be destroyed.
```
